### PR TITLE
docs: americanize the spelling of words in sourced markdown

### PR DESCRIPTION
**Pull request by xginn8**
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

Resolves: #   
Change-type: major|minor|patch   
Depends-on: <url>   
See: <url>   

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

Thank you very much for your contribution!

### CODE REVIEW

**Review by srlowe — APPROVED** [review]

File: doc/cli.markdown [hunk 1]
```diff
@@ -2509,7 +2509,7 @@ any) defined at the overall project root will be used to filter files and
 subdirectories other than service subdirectories. It will not have any effect
 on service subdirectories, whether or not a service subdirectory defines its
 own .dockerignore file. Multiple .dockerignore files are not merged or added
-together, and cannot override or extend other files. This behavior maximises
+together, and cannot override or extend other files. This behavior maximizes
```
Comment (pdcastro) [inline doc/cli.markdown:2512]: `cli.markdown` is auto generated though. You need to search and find where this text comes from in source code (Typescript or Javascript), then run `npm run build` to refresh `cli.markdown` and commit.

**Comment by srlowe** [comment]
Oops - I quickly approved this, but then realized that the change was made in the wrong place.  Have converted to draft to stop merge.  Will discuss with @xginn8 on FD.

**Comment by pdcastro** [comment]
> Have converted to draft to stop merge.

No worries - the tests fail anyway with an "uncommitted changes" error when this happens. :-)

File: doc/cli.markdown (hunk 1 above)
Comment (xginn8) [reply doc/cli.markdown:2512]: good catch, thanks!

**Comment by xginn8** [comment]
@balena-ci I self-certify!

**Pull request merged**